prac_05/color_names.py

```diff
@@ -0,0 +1,14 @@
+COLOR_NAMES = {"turquoise": "#40e0d0", "yellowgreen": "#9acd32", "salmon": "#fa8072", "saddlebrown": "#8b4513"}
+
+color_length = [len(color) for color in COLOR_NAMES]
+
+for color in COLOR_NAMES:
```

Comment:
why is this printing all the colours with their hex value and then having the user input the colour name after